inclusive-design/wecount.inclusivedesign.ca
# Svg on high contrast

## Description

To resolve https://github.com/inclusive-design/wecount.inclusivedesign.ca/issues/136 and https://github.com/inclusive-design/wecount.inclusivedesign.ca/issues/236

This pull request is based off the pull request for adding UIO (https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/230), which should be reviewed and merged first.

Use [this link](https://github.com/TeddTech/wecount.inclusivedesign.ca/compare/add-uio...cindyli:svg-on-highContrast) to view the difference between this pull request and [the pull request](https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/230) it's based off.

## Steps to test

1. Open UIO and select different high contrast themes;
2. SVGs on the header, footer, news and views pages should adapt to high contrast colors.
